packages/tracing/test/browser/browsertracing.test.ts

```diff
@@ -210,12 +210,26 @@ describe('BrowserTracing', () => {
       const name = 'sentry-trace';
       const content = '126de09502ae4e0fb26c6967190756a4-b6e54397b12a2a0f-1';
       document.head.innerHTML = `<meta name="${name}" content="${content}">`;
+      const startIdleTransaction = jest.spyOn(hubExtensions, 'startIdleTransaction');
+
       createBrowserTracing(true, { routingInstrumentation: customRoutingInstrumentation });
-      const transaction = getActiveTransaction(hub) as IdleTransaction;
 
-      expect(transaction.traceId).toBe('126de09502ae4e0fb26c6967190756a4');
-      expect(transaction.parentSpanId).toBe('b6e54397b12a2a0f');
-      expect(transaction.sampled).toBe(true);
+      // we match on the calls themselves (rather than calling .toHaveBeenCalledWith()) because that method requires all
+      // of the arguments be supplied, when we really only care about the transaction context which gets passed
+      expect(startIdleTransaction.mock.calls).toEqual(
```

Comment:
This spy is local to this test, so I'm pretty sure doing `mock.calls[0][0]` instead would be enough, as current assertion is not too readable imo.

Reply:
`expect.any()` to the rescue!